src/App/Pages/Vault/AttachmentsPage.xaml.cs

```diff
@@ -53,10 +53,7 @@ protected override void OnDisappearing()
 
         private void Save_Clicked(object sender, EventArgs e)
         {
-            if (DoOnce())
-            {
-                _vm.SubmitAsyncCommand.Execute(null);
-            }
+            _vm.SubmitAsyncCommand.Execute(null);
         }
```

Comment:
Sorry I missed it before but you can remove this method and use `Command` in xaml to bind to `SubmitAsyncCommand`